README.rst

```diff
@@ -72,6 +72,20 @@ or
 
     python setup.py install
 
+
+New to howdoi ? 
+----------------
+
+::
+
+    howdoi use howdoi
```

Comment:
maybe just howdoi howdoi. we can support all the different variants but this one is the shortest, and also the funniest.